src/org/opensolaris/opengrok/configuration/RuntimeEnvironment.java

```diff
@@ -96,6 +96,7 @@
 import org.opensolaris.opengrok.web.Util;
 
 import static java.nio.file.FileVisitResult.CONTINUE;
+import java.nio.file.InvalidPathException;
```

Comment:
Breaks checkstyle's import order option: <http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderOption.html#UNDER>

I will have to work on enabling that checkstyle...